src/pybind/mgr/dashboard/services/ceph_service.py

```diff
@@ -210,6 +211,10 @@ def send_command(srv_type, prefix, srv_spec='', **kwargs):
         :raises TimedOut: See rados.make_ex
         :raises ValueError: return code != 0
         """
+        if not CephService.get_service_map(srv_type):
+            raise DashboardException(msg=f'{srv_type} service might be down', http_status_code=500,
+                                     code='send_command_no_service')
```

Comment:
Are we sure we want to call a method that invokes the mgr API `2 x number_of_daemons` (this will grow with the cluster size) every time we send a command (with all the daemon metadata)? 

https://github.com/rhcs-dashboard/ceph/blob/b1a840e3d2587121dced99147321c626628006d6/src/pybind/mgr/dashboard/services/ceph_service.py#L41-L62

Additionally, will the reported segfault only affect the Dashboard code, or anyone else using the `mgr.send_command()` call? If that's the case, why not fixing this in the Ceph mgr side (either at the Python or C++ side)?